documentation/paper.py

```diff
@@ -3,7 +3,7 @@
 def configure(context):
     context.stage("documentation.plots.data.hts_comparison")
     #context.stage("documentation.plots.theory.sampling_error")
-    context.stage("documentation.plots.monte_carlo")
+    # context.stage("documentation.plots.monte_carlo")
```

Comment:
Activate again, actually, can we maybe add a config switch that activates this or not? Then we can generate the paper plots without during the MC experiment